wiki/Ranking_Criteria/Song_Content_Rules/en.md

```diff
@@ -6,7 +6,7 @@ _Main page: [Ranking Criteria](/wiki/Ranking_Criteria)_
 With a wide variety of music available, most tracks will be fine for use in osu!, providing that they are not:
 
 - Intensely explicit, suggestive, or otherwise extremely confrontational in regards to shocking, obscene or highly sexual content
-- Glorifying, promoting or otherwise encouraging suicide, murder, or self-harm
+- Glorifying, promoting or otherwise encouraging drug abuse, suicide, murder, or self-harm
```

Comment:
¿drug usage?

Reply:
"drug usage" itself is too broad imo and can be construed as including maps like: https://osu.ppy.sh/beatmapsets/5823#osu/27993 

"drug abuse" in my eyes is a step further and more appropriate in this context